src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -305,7 +308,8 @@ def delete_vcs_repository(cls, repo_name) -> str:
             response = requests.delete(url, headers=headers)
 
             if response.status_code != 204:
-                cls.set_up_errors.append(f"Failed to delete the repository. Status code: {response.status_code}, Response: {response.text}")
+                if not silent:
+                    cls.set_up_errors.append(f"Failed to delete the repository. Status code: {response.status_code}, Response: {response.text}")
                 return ""
```

Comment:
The return value is not documented in the docstring. It look like you're returning the name of the repo upon successful deletion or the empty string if it fails.

In the case that the repo does not exist, is that considered a success or failure? Should it return an empty string or not.

Reply:
This will be addressed in [CRAYSAT-2007](https://jira-pro.it.hpe.com:8443/browse/CRAYSAT-2007)